pkg/driver/helmdriver_test.go

```diff
@@ -105,6 +109,15 @@ func TestIsConfigChanged(t *testing.T) {
 //
 // Helpers
 //
+func createNewHelm(t *testing.T) (error, *helmDriver) {
+	fakeRestClient := fakerest.RESTClient{
+		GroupVersion: api.GroupVersion,
+	}
+	secretAuth, err := auth.NewECRSecret(&fakeRestClient)
+	require.NoError(t, err)
+	helm, err := NewHelm(logr.Discard(), secretAuth)
+	return err, helm
+}
```

Comment:
```suggestion
func createNewHelm(t *testing.T) (*helmDriver,error) {
	fakeRestClient := fakerest.RESTClient{
		GroupVersion: api.GroupVersion,
	}
	secretAuth, err := auth.NewECRSecret(&fakeRestClient)
	require.NoError(t, err)
	helm, err := NewHelm(logr.Discard(), secretAuth)
	require.NoError(t, err)
	return helm, err
}
```
Putting the error before the helm driver is weird in golang speak.

Reply:
changed and fixed tests to match